test/cdk-test-integration/functions/readFunction.js

```diff
@@ -0,0 +1,17 @@
+var AWS = require('aws-sdk');
+var TABLE_NAME = process.env.HELLO_TABLE_NAME;
+exports.handler = async (event, context) => {
+  const dbClient = new AWS.DynamoDB({ region: "us-east-1" });
+    try{
+      var params = {
+        TableName: TABLE_NAME,
+        Key: {
+          'id': { S: 'id'},
+        }
+      };
+      await dbClient.getItem(params).promise();
+      console.log('success')
```

Comment:
Can you print out the item that was retrieved? Better yet, validate that the value stored in the table was a value that was expected to be there.